WordPress/Classes/Services/ReaderTopicService+Interests.swift

```diff
@@ -1,6 +1,19 @@
 import Foundation
 import WordPressKit
 
+// MARK: - ReaderInterestsService
+
+/// Protocol representing a service that retrieves a list of interests stored remotely
```

Comment:
One may wonder that `stored remotely` means that this is the user's list of interests.

This just returns the list of the interests that the user can follow, right? If so, I guess we can update the comment to better reflect that.

Reply:
Done! 263b50fa21ab548902d78b95a44f46f3559121c5